app/scripts/components/alerts.component.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+angular.module('openshiftConsole')
+  .component('alerts', {
+    bindings: {
+      // Map (or array) of alert objects, each with properties
+      //   type:     'error', 'warning', 'success', or 'info' (default)
+      //   message:  the message to display
+      //   links:    an array of link objects (optional, see below)
+      //   onClose:  callback function when the user dismisses the alert (optional)
+      //
+      // Each link can have the following properties:
+      //   href:     link URL (optional if onClick is set)
+      //   label:    the link text
+      //   onClick:  handler to call when the link if clicked (optional)
+      //             If onClick returns true, dismisses the alert
+      alerts: '=',
+      // Filter function (optional).
+      filter: '=?',
+      animateSlide: '=?',
+      hideCloseButton: '=?',
+      toast: '=?'
+    },
+    templateUrl: 'views/_alerts.html',
+    controller: function() {
```

Comment:
We have a convention for using named functions for controllers for components.  That said, I prefer inline anonymous, I don't think we gain anything out of the named functions other than the need for IIFE's around all of these files.  

@spadgett I think may have a preference for the named function however, so I defer to his call on it.